cms/djangoapps/contentstore/views/component.py

```diff
@@ -195,6 +195,9 @@ def container_handler(request, usage_key_string):
 
             # Get the status of the user's clipboard so they can paste components if they have something to paste
             user_clipboard = content_staging_api.get_user_clipboard_json(request.user.id, request)
+            library_block_types = [problem_type['component'] for problem_type in LIBRARY_BLOCK_TYPES]
```

Comment:
A lot of the code in this PR is in support of the library_sourced block, which it turns out that we will be removing. Do you plan on cleaning of that up in this PR, or in a follow-up?

Reply:
@connorhaugh could you at least take the library_sourced parts out of this PR so it's easier to review?

Reply:
done here https://github.com/openedx/edx-platform/pull/33257